controllers/topics.controller.js

```diff
@@ -28,4 +28,22 @@ exports.getArticles = (req, res, next) => {
     })
 }
 
+exports.getComments = (req, res, next) => {
+    const article_id = req.params.article_id;
+    const promises = [fetchComments(article_id)];
+
+    
+    if(article_id) {
+        promises.push(fetchArticleId(article_id));
+    }
+
```

Comment:
Is this conditional logic needed? The article_id should be there - if not it won't get to this endpoint :)